src/Components/Browser.JS/src/BootCommon.ts

```diff
@@ -37,3 +37,13 @@ interface BootJsonData {
   jsReferences: string[];
   linkerEnabled: boolean;
 }
+
+// calls Blazor.start() if the script was added with <script src="..." start></script>
+export function autoStartIfApplicable(start: (options?: any)=> Promise<void>) {
+  if (document &&
+    document.currentScript &&
+    document.currentScript.hasAttribute("start") &&
+    document.currentScript.getAttribute("start") !== "false") {
+      start();
+  }
```

Comment:
LMK if there's a more idiomatic way to do this. i is noob

Reply:
The chain of `if`s looks fine. However:

 * Why take a callback as a param, given that you're not really making any use of its args or return value? It's declared as accepting `options` and returning a promise, but the usage here supports neither. Instead maybe replace `autoStartIfApplicable` with `shouldAutoStart(): boolean`, and have the upstream code do what it wants with the bool result.
 * Nit: quotes should be single quotes. Javier is adding tslint in his separate PR.